alerts/etcdNoLeader.md

```diff
@@ -0,0 +1,44 @@
+# etcdNoLeader
+
+## Meaning
+
+This alert is triggered when etcd cluster does not have a leader for more than 1
+minute.
+
+## Impact
+
+When there is no leader, Kubernetes and OpenShift APIs will not be able to work
+as expected and cluster cannot process any writes, and any write requests are
```

Comment:
or reads as linearized reads are the default.

Reply:
Done, thanks!